presets/mainnet/network.yml

```diff
@@ -50,6 +50,8 @@ fileDatabaseBatchSize: 100
 minProofSize: 0
 syncTimeout: 5m
 maxChildNamespaces: 100
+unfinalizedBlocksDuration: 0m
+votingUnfinalizedBlocksDuration: 10m
```

Comment:
should probably be votingUnfinalizedBlocksDuration => nonVotingUnfinalizedBlocksDuration?